backend/src/seeds/factory/creation.ts

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 
-import { createPendingCreation, confirmPendingCreation } from '@/seeds/graphql/mutations'
+import { adminCreateContribution, confirmPendingCreation } from '@/seeds/graphql/mutations'
```

Comment:
please do not mix Rights/Rules with method names, especially when with replacement of the elopage process the user will capture his contributions himself and the admin will only confirm them